bin/schema-registry-run-class

```diff
@@ -63,7 +63,7 @@ fi
 
 # Memory options
 if [ -z "$SCHEMA_REGISTRY_HEAP_OPTS" ]; then
-  SCHEMA_REGISTRY_HEAP_OPTS="-Xmx256M"
+  SCHEMA_REGISTRY_HEAP_OPTS="-Xmx512M"
```

Comment:
@ewencp, yeah I was getting OOMEs. I didn't test whether a smaller value would work or not. Want me to test a smaller value, like `384`?